core/commands/log.go

```diff
@@ -34,7 +34,8 @@ system (not just for the daemon logs, but all commands):
 	Subcommands: map[string]*cmds.Command{
 		"level": logLevelCmd,
 		"ls":    logLsCmd,
-		"tail":  logTailCmd,
+		// FIXME: Re-enable `ipfs log tail` command. Tracked in https://github.com/ipfs/kubo/issues/9245
+		// "tail":  logTailCmd,
```

Comment:
linting is failing because `logTailCmd` is defined and unused, we can remove the function, or leave it and just add messaging.. lmk